src/commands/set.js

```diff
@@ -80,19 +83,26 @@ const setBoolean = (variable, path) => value => {
 		liskAPIInstance.setTestnet(newValue);
 	}
 
-	return attemptWriteToFile(variable, value);
+	if (variable === 'ssl') {
+		liskAPIInstance.setSSL(newValue);
+	}
+
+	return attemptWriteToFile(variable, value, path);
 };
 
 const setString = (variable, path) => value => {
 	path.reduce(setNestedConfigProperty(value), config);
-	return attemptWriteToFile(variable, value);
+	return attemptWriteToFile(variable, value, path);
 };
 
 const handlers = {
 	json: setBoolean('json', ['json']),
 	name: setString('name', ['name']),
 	pretty: setBoolean('pretty', ['pretty']),
-	testnet: setBoolean('testnet', ['liskJS', 'testnet']),
+	'liskJS.testnet': setBoolean('testnet', ['liskJS', 'testnet']),
+	'liskJS.ssl': setBoolean('ssl', ['liskJS', 'ssl']),
+	'liskJS.node': setString('node', ['liskJS', 'node']),
+	'liskJS.port': setString('port', ['liskJS', 'port']),
 };
```

Comment:
We shouldn't need these handlers anymore. They were put in place because the path and identifier weren't uniformly related.

Reply:
They still serve the purpose of exposing and dividing into string or boolean value. I'd prefer to keep it for now.